optimum/exporters/openvino/__main__.py

```diff
@@ -23,10 +23,21 @@
 from optimum.exporters import TasksManager
 from optimum.exporters.onnx import __main__ as optimum_main
 from optimum.exporters.onnx.base import OnnxConfig, OnnxConfigWithPast
+
+
+try:
+    from optimum.exporters.onnx.constants import SDPA_ARCHS_ONNX_EXPORT_NOT_SUPPORTED
+except ImportError:
```

Comment:
we can also check the optimum version with [`is_optimum_version`](https://github.com/huggingface/optimum-intel/blob/a30e337542c5d81c1ba0c6545fa0a5651f046dec/optimum/intel/utils/import_utils.py#L179) (might be useful in the future in case something breaks)

Reply:
refactored